packages/command-palette/test/commands.tsx

```diff
@@ -0,0 +1,257 @@
+	resetSshSftpPassword: [ '/hosting-config/:site#sftp-credentials', 'atomicAdmin' ],
+	openJetpackStats: [ '/wp-admin/admin.php?page=stats' ],
+	openActivityLog: [ '/activity-log/:site', 'wpcomP2' ],
+	openJetpackBackup: [ '/backup/:site', 'wpcomAdminP2' ],
+	viewSiteMonitoringMetrics: [ '/site-monitoring/:site', 'atomicAdmin' ],
+	openGitHubDeployments: [ '/github-deployments/:site', 'atomicAdmin' ],
+	openPHPLogs: [ '/site-monitoring/:site/php', 'atomicAdmin' ],
+	openWebServerLogs: [ '/site-monitoring/:site/web', 'atomicAdmin' ],
+	manageStagingSites: [ '/hosting-config/:site#staging-site', 'atomicAdmin' ],
+	changePHPVersion: [ '/hosting-config/:site#web-server-settings', 'atomicAdmin' ],
+	changeAdminInterfaceStyle: [ '/hosting-config/:site#admin-interface-style', 'atomicAdmin' ],
+	addNewPost: [ '/wp-admin/post-new.php', 'posts' ],
+	managePosts: [ '/wp-admin/edit.php', 'posts' ],
+	viewMediaUploads: [ '/wp-admin/upload.php', 'media' ],
+	uploadMedia: [ '/wp-admin/media-new.php', 'media' ],
+	managePages: [ '/wp-admin/edit.php?post_type=page', 'pages' ],
+	addNewPage: [ '/wp-admin/post-new.php?post_type=page', 'pages' ],
+	manageComments: [ '/wp-admin/edit-comments.php', 'comments' ],
+	manageThemes: [ '/wp-admin/themes.php', 'themesP2' ],
+	installTheme: [ '/wp-admin/theme-install.php', 'themesJetpack' ],
+	managePlugins: [ '/wp-admin/plugins.php', 'pluginsP2' ],
+	installPlugin: [ '/wp-admin/plugin-install.php', 'pluginsJetpack' ],
+	changePlan: [ '/plans/:site', 'adminStagingP2' ],
+	manageMyPlan: [ '/plans/my-plan/:site', 'adminStagingP2' ],
+	manageUsers: [ '/wp-admin/users.php', 'users' ],
+	addNewUser: [ '/wp-admin/user-new.php', 'users' ],
+	addSubscribers: [ '/subscribers/:site#add-subscribers', 'admin' ],
+	manageSubscribers: [ '/subscribers/:site', 'admin' ],
+	downloadSubscribers: [
+		'https://dashboard.wordpress.com/wp-admin/index.php?page=subscribers&blog=:siteId&blog_subscribers=csv&type=all',
+		'admin',
+	],
+	import: [ '/import/:site', 'admin' ],
+	openWooCommerceSettings: [ '/woocommerce-installation/:site', 'wpcomAdminP2' ],
+	manageSettingsGeneral: [ '/wp-admin/options-general.php', 'admin' ],
+	manageSettingsWriting: [ '/wp-admin/options-writing.php', 'admin' ],
+	manageSettingsReading: [ '/wp-admin/options-reading.php', 'admin' ],
+	manageSettingsDiscussion: [ '/wp-admin/options-discussion.php', 'admin' ],
+	manageSettingsNewsletter: [ '/wp-admin/admin.php?page=jetpack#/newsletter', 'admin' ],
+	manageSettingsPodcast: [ '/settings/podcasting/:site', 'admin' ],
+	sendFeedback: [ '/help' ],
+};
+
+describe( 'COMMANDS', () => {
+	it( 'should be correctly defined', () => {
+		for ( const [ command, expectedResults ] of Object.entries( expectedCommandsResults ) ) {
+			const expectedPath = expectedResults[ 0 ];
+			expect( getNavigationPath( COMMANDS[ command ] ) ).toEqual( expectedPath );
+
+			const expectedSiteFilter = expectedResults[ 1 ];
+			switch ( expectedSiteFilter ) {
```

Comment:
I'm not too keen on this switch. Could we include the filters in the `expectedCommandResults`?

Something like:
```
const expectedCommandsResults = {
	// <COMMAND_NAME>: [ <PATH>, <TYPE_OF_SITE_FILTER> ]
         .
         .
         .
	clearCache: [ '/hosting-config/:site#cache',  { siteType: 'atomic', capability: 'manage_options' } ],
```

Then we could check without needing the giant switch. WDYT?

Reply:
https://github.com/Automattic/wp-calypso/pull/89040/commits/97ff8d4b5e56ba235191a55d7ad3a9a05413c465